benchmarking/runs/concentrate_gpa_run.py

```diff
@@ -61,6 +76,85 @@ def concentrate_gpa_run():
             "Minimum Gini Index for GPA =>",
             Simulation.average_metric(simulation_outputs, "MinimumGiniIndex"),
         )
+        print(
+            "Runtimes =>",
+            Simulation.average_metric(simulation_outputs, "runtimes"),
```

Comment:
nit; 

technically better to use `Simulation.KEY_RUNTIMES` instead of the string